every_election/apps/elections/management/commands/assign_election_to_divisionset.py

```diff
@@ -0,0 +1,83 @@
+"""
+This management command assigns elections from election groups to a given divisionset's divisions.
+
+This is usually used when we're processing Community Governance Reviews as decscribed in this issue:
+
+https://github.com/DemocracyClub/EveryElection/issues/570.
+
+python manage.py assign_election_to_divisionset <election_ids> <divisionset_id>
+
+example:
+python manage.py assign_election_to_divisionset local.broxtowe.2023-05-04 local.broxtowe.2024-05-02 794
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from elections.models import Election
+from organisations.models import OrganisationDivision
+
+
+class Command(BaseCommand):
+    help = "Tries to assign elections from election groups to a given divisionset's divisions"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "election_group_ids",
+            action="store",
+            nargs="+",
+            help="space separated list of election_ids",
+        )
+        parser.add_argument(
+            "divset_id",
+            action="store",
+            type=int,
+            help="The divisionset_id of the divisionset",
+        )
+
+    def handle(self, *args, **options):
+        divset_id = options.get("divset_id")
+        election_group_ids = options.get("election_group_ids", [])
```

Comment:
Given these arguments are required, you should be able to do

```suggestion
        divset_id = options["divset_id"]
        election_group_ids = options["election_group_ids"]
```

In fact, it would be preferable to to throw `KeyError` if we are here and one of those args is not defined.